service/mapper/transitions.go

```diff
@@ -269,9 +271,124 @@ func (t *Transitions) ResumeIndexing(s *State) error {
 	// then proceed with the first non-indexed block and index values
 	s.height = last + 1
 
-	// At this point, we should be able to start indexing the chain data for
+	// At this point, we should be able to start indexing the register data for
 	// the next height.
-	s.status = StatusIndex
+	s.status = StatusUpdate
+	return nil
+}
+
+// UpdateTree gets all trie updates and stores it in the state
+func (t *Transitions) UpdateTree(s *State) error {
+	if s.status != StatusUpdate {
+		return fmt.Errorf("invalid status for updating tree (%s)", s.status)
+	}
+	log := t.log.With().Uint64("height", s.height).Logger()
+	// grab updates and move on to next state
+	updates, err := t.updates.AllUpdates()
+	if errors.Is(err, archive.ErrUnavailable) {
+		time.Sleep(t.cfg.WaitInterval)
+		log.Debug().Msg("waiting for next trie update")
+		return nil
+	}
```

Comment:
No, since the source of the data is the GCP bucket, we want to keep retrying until the file is found. This is so the archive node can stay live while we try and fix it manually.